metabase/metabase
# Fix links in admin settings to work on subpath

Closes https://github.com/metabase/metabase/issues/33910

### Description

We have two broken links:
- connection pool details in the admin settings didn't respect deployment on the subpath
- permissions graph was broken

My initial approach was to patch `ExternalLink` component to support subpath deployment, but it turned out it's the only use case, so I just handled it manually

This PR fixes those links

### How to verify

1. permissions graph
Go to admin settings of EE app
-`/admin/permissions/application`
- update any permissions
- try to save - there shouldn't be an error in the network tab of devtools

2. connection pool
- deploy on subpath 
- go to admin settings -> troubleshooting -> click on Connection Pool Details link -> browser should download json
<details><summary>subpath deployment instructions</summary>
<p>

1. run your local instance (frontend and backend separately), go to the question and download results (csv, json, xlsx) - make sure they show correct result
2. go to admin settings -> change metabase URL from `localhost:3000` to `localhost:9090/metabase/`
3. stop your frontend server, keep backend server running
4. build js files with `yarn build`, open `localhost:3000` and make sure js files are loaded
5. clone https://github.com/iethree/metabase-subpath-proxy: e.g. `cd .. && git clone git@github.com:iethree/metabase-subpath-proxy.git`
6. go to the  e.g. `cd ./metabase-subpath-proxy` and run `docker-compose up` (or `docker compose up`)
7. now open `localhost:9090/metabase/` - notice `/` at the end of the line, it's required

</p>
</details> 

### Demo

https://www.loom.com/share/3558113038fc4a4db330fcb33b76770a

### Checklist

~- [ ] Tests have been added/updated to cover changes in this PR~
